platforms/cuda/src/CudaTorchKernels.cpp

```
@@ -92,96 +92,156 @@ void CudaCalcTorchForceKernel::initialize(const System& system, const TorchForce
     CUmodule program = cu.createModule(CudaTorchKernelSources::torchForce, defines);
     copyInputsKernel = cu.getKernel(program, "copyInputs");
     addForcesKernel = cu.getKernel(program, "addForces");
+    const std::string useCUDAGraphsString = force.getProperty("useCUDAGraphs");
+    if (useCUDAGraphsString == "true")
+        useGraphs = true;
+    else if (useCUDAGraphsString == "false" || useCUDAGraphsString == "")
+        useGraphs = false;
+    else
+        throw OpenMMException("TorchForce: invalid value of \"useCUDAGraphs\"");
+    this->warmupSteps = 1;
+    if (useGraphs) {
+        const std::string warmupStepsString = force.getProperty("CUDAGraphWarmupSteps");
+        if (!warmupStepsString.empty())
+            this->warmupSteps = std::stoi(warmupStepsString);
```

We should throw an exception if it's less than 1.